### ISSUE TITLE
Eclipse importer exception: missing property '_type'  

### ISSUE THREAD

**Issue by nmatra** [ossmeter/ossmeter#84]
I am trying to import a project using the Eclipse importer but I get the following error:

``` Java
Unexpected token (END_OBJECT), expected FIELD_NAME: missing property '_type' that is to contain type id  (for class org.ossmeter.repository.model.Company)
 at [Source: java.io.StringReader@133f5494; line: 1, column: 2563] (through reference chain: org.ossmeter.repository.model.eclipse.EclipseProject["companies"])
```

I have tried this with multiple Eclipse projects, but I get the same behaviour.  Moreover, I tried this with the version running on the servers and locally. 

 For example trying to import papyrus I  used the following URL: https://projects.eclipse.org/projects/modeling.mdt.papyrus

Given that this used to work until recently, this might be due to a change in Eclipse’s API?

**Comment by jdirocco** [comment]
This seems to be similar to issue #31 . In the running version, If a project contains some companies the importer fails otherwise the import method correctly works.

Using org.ossmeter.platform.importer.test project, I have import all eclipse projects by importAll method. Many project contains companies. None of these generate this problem when try to import.
Project papyrus also is correctly imported by 

```
EclipseProjectImporter im;
im = new EclipseProjectImporter();
importProjectByUrl("https://projects.eclipse.org/projects/modeling.mdt.papyrus", platform);
```

I don't think that this issue depends by the last changes. I'm going to investigate on that problem.

**Comment by nmatra** [comment]
I think this problem is related to Pongo. I am looking at this as well.

**Comment by nmatra** [comment]
Fixed in f95b32330dbf977c13f05abdf4b0a21ac1578a56
